src/App.jsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import WeatherRequest from './WeatherRequest';
+
+const API_KEY = '39023aff7ae25b6655f72f0935e31fc6';
 
 function App() {
+  const [currentWeather, setCurrentWeather] = useState();
+  const [hourlyWeather, setHourlyWeather] = useState();
+  const [dailyWeather, setDailyWeather] = useState();
+
+  const coordinates = {
+    latitude: '45.50742',
+    longitude: '-122.68984',
+  };
+
+  WeatherRequest({
+    coordinates,
+    setCurrentWeather,
+    setHourlyWeather,
+    setDailyWeather,
+    API_KEY,
+  });
```

Comment:
I didn't know you could pass the useState functions as props? Is this valid?

Reply:
You're right, I'll take a look at refactoring that.

Reply:
In the refactor I replaced the `useState` calls with `addCurrentWeather`, etc. to contain `useState` within the parent.